File: Source/Jekov.Nevix/Common/Jekov.Nevix.Common.ViewModels/UserRegisterViewModel.cs
namespace Jekov.Nevix.Common.ViewModels
{
    using System.ComponentModel.DataAnnotations;
    using System.Runtime.Serialization;

    [DataContract]
    public class UserRegisterViewModel
    {
        [Required]
        [DataMember(Name = "email")]
        public string Email { get; set; }

        [Required]
        [DataMember(Name = "password")]
        public string Password { get; set; }

        [Required]
        [DataMember(Name = "confirmPassword")]
        public string ConfirmPassword { get; set; }
    }
}

File: Source/Android/Nevix/nevix/src/main/java/com/antonyjekov/nevix/viewmodels/UserRegisterViewModel.java
package com.antonyjekov.nevix.viewmodels;

public class UserRegisterViewModel extends UserLoginViewModel {
    private String confirmPassword;

    public UserRegisterViewModel(String email, String pass, String confirmPass){
        super(email, pass);

        this.confirmPassword = confirmPass;
    }

    public String getConfirmPassword() {
        return confirmPassword;
    }

    public void setConfirmPassword(String confirmPassword) {
        this.confirmPassword = confirmPassword;
    }
}
